variables.mdx

```diff
@@ -1,6 +1,6 @@
 ---
 title: Variables
-description: "Variable templates"
+description: "Variable templates for tests"
```

Comment:
```suggestion
description: "Variable templates in tests"
```

to be honest not 100% sure about this proposition myself, but somehow "in" FEELS more correct? 😄